### PR TITLE
[WIP] #2070 Fix PDP Url Path For Grid Settings

### PR DESCRIPTION
**Pull request by michael-alade**
**Resolves**
#2070 

**Manual Test**
- Add a new product with a variant
- Do not Publish.
- Go to Grid and Select the product
- Click product item on grid setting view.
- The page should show.

### CODE REVIEW

File: imports/plugins/included/product-variant/client/templates/products/productSettings/productSettings.js [hunk 1]
```diff
@@ -67,15 +67,85 @@ Template.productSettings.helpers({
   }
 });
 
-Template.productSettingsGridItem.helpers({
-  displayPrice: function () {
+// Template.productSettingsGridItem.helpers({
```
Comment (zenweasel) [inline imports/plugins/included/product-variant/client/templates/products/productSettings/productSettings.js]: Let's not have commented out code. That's what source control is for. Let's remove it or leave it.

**Comment by zenweasel** [comment]
Tested, Seems to work as described.

File: imports/plugins/included/product-variant/client/templates/products/productSettings/productSettings.js (hunk 1 above)
Comment (michael-alade) [inline imports/plugins/included/product-variant/client/templates/products/productSettings/productSettings.js]: ok.

**Comment by michael-alade** [comment]
@zenweasel changes have been made to the files you reviewed.

**Pull request merged**